interface/main/tabs/main.php

```diff
@@ -431,6 +431,9 @@ function showUpdaterNotifications(type, title, message) {
             message: message,
         });
     }
+
+
```

Comment:
Trim these please

Reply:
I don't mind a little white space in my code personally.  Especially between logical sections and jumps between script types.